composer.json

```diff
@@ -25,7 +25,8 @@
         "source": "https://github.com/ramsey/uuid"
     },
     "require": {
-        "php": ">=5.4"
+        "php": ">=5.4",
+        "paragonie/random_compat": "^2.0"
```

Comment:
Best to allow `^1.0|^2.0` otherwise will will conflict with just about everything atm.